tests/Feature/UpdateOutgoingLettersTest.php

```diff
@@ -27,18 +27,17 @@ public function guest_cannot_update_letters()
     /** @test */
     public function user_can_update_outgoing_letter_in_database()
     {
-        $letter = factory(OutgoingLetter::class)->create();
-        $this->be(factory(User::class)->create());
-        
-        $new_outgoing_letter = factory(OutgoingLetter::class)->make();
+        $this->be($user = factory(User::class)->create());
+        $letter = factory(OutgoingLetter::class)->create(['creator_id' => $user->id]);
+        $new_outgoing_letter = factory(OutgoingLetter::class)->make(['creator_id' => $user->id]);
```

Comment:
This does not make sense to override `creator_id` for making new letter. We do not rely on `creator_id` field as an input. While, updating the outgoing letter you don't even need to send `creator_id` field, infact you should NOT send this field for this "happy path" test case.

You can write a seperate test case, assuming user tries to send a new `creator_id` and you can then assert that there is no effect of this input. `creator_id` remains same.